client/cmd/import.go

```diff
@@ -37,8 +37,8 @@ func init() {
 
 var importCmd = &cobra.Command{
 	Use:    "import",
-	Short:  "Evaluate external format and import as Alda source code",
-	Long: `Evaluate external format and import as Alda source code
+	Short:  "Import Alda source code from MusicXML",
+	Long: `Import Alda source code from MusicXML
```

Comment:
Suggestion: "Import Alda source code from other formats"

While it's true that MusicXML is currently the only supported format, I think the help text should describe the general intent of this command, which is to potentially support various formats in the future.